package.json

```diff
@@ -27,6 +27,7 @@
     "babel-plugin-add-module-exports": "1.0.2",
     "babel-plugin-lodash": "3.3.4",
     "bcrypt": "3.0.6",
+    "blockstack": "^19.1.0",
```

Comment:
You said integrating this `blockstack` library might not be mandatory. What would be the alternative? If it's simple, then let's do that.

Reply:
I had a look but it is a bit more complicated.